usr/lib/sticky/common.py

```diff
@@ -259,7 +254,7 @@ def flush(self):
 
     def new_group(self, group_name):
         if group_name in self.notes_lists:
-            if not confirm(_("Overwrite Existing Group"), _("There is already a group named '%s'. This action will overwrite it. Continue anyway?") % group_name, self.window):
+            if not confirm(_("Overwrite Existing Group"), _(f"There is already a group named '{group_name}'. This action will overwrite it. Continue anyway?{self.window}")):
```

Comment:
Any strings marked for translation (`_(...)`)will need to stay in the traditional format, as these changes will break the existing translations, and I'm not even sure gettext supports f string format in the first place.